src/compute_horde_prompt_gen/utils.py

```diff
@@ -8,22 +8,34 @@
 
 def clean_line(line: str) -> str:
     line = line.strip()
+    head, sep, tail = line.partition("<|")
+    if head:
+        line = head.strip()
+    else:
+        # if we started with a tag we assume that inside we find our prompt
+        line = tail.partition("|>")[2].partition("<|")[0].strip()
     # remove list numbering if present
     line = re.sub(r"^\s*\d+\.?\s*", "", line)
+    # strip quotations
+    line = line.strip("\"'")
```

Comment:
Won't that mess up lines, where a quote mark is at the start/end of the line, and the other quote in the middle of the line? The strip will only remote the quote at the start/end, not the quote in the middle.

Reply:
we don't care - we care ONLY about the lines that are single line prompt - and ends with `?` - if result is broken or "not perfect" - it will be ignored